storage/tests/system.py

```diff
@@ -547,6 +547,32 @@ def test_compose_replace_existing_blob(self):
         composed = original.download_as_string()
         self.assertEqual(composed, BEFORE + TO_APPEND)
 
+    @unittest.skipUnless(USER_PROJECT, 'USER_PROJECT not set in environment.')
```

Comment:
I'm beginning to think we should just have a `class TestWithUserProject(unittest.TestCase):` that has a `skipUnless` applied to the whole class.

Reply:
We would have to replicate all the varioius `setUp` / `tearDown` bits I'm currently reusing.